Makefile

```diff
@@ -67,7 +66,7 @@ docker-make-install:
 		make install
 
 .PHONY: e2e_test
-e2e_test:
+e2e_test: build lint test dep
```

Comment:
`s/dep/check-dependencies/` — we'd like to know when deps are updated but not checked-in
also add `licence-check` to this step.

Reply:
OK, probably no `licence-check`, sorry, it's different image.

Reply:
Regarding changing `dep` to checking `check-dependencies`, it would require a change to the presubmit, so let's leave it for a follow-up.